python/pyarrow/_dataset.pyx

```diff
@@ -1963,6 +1963,100 @@ cdef class FragmentScanOptions(_Weakrefable):
         except TypeError:
             return False
 
+cdef class CacheOptions(_Weakrefable):
+    """
+    Cache options for a pre-buffered fragment scan.
+    Parameters
```

Comment:
```suggestion

    Parameters
```

Reply:
Pretty sure I did this. Not sure why github hasn't recognised the comment as `outdated`.